spec/features/user_management_spec.rb

```diff
@@ -0,0 +1,60 @@
+require './spec/factories/user'
+
+feature 'User sign up' do
+
+
+  scenario 'I can sign up as a new user' do
+    expect { sign_up }.to change(User, :count).by(1)
+    expect(page).to have_content('Welcome, pip')
+    expect(User.first.email).to eq('pip@example.com')
+  end
+
+  scenario 'requires a matching comfirmation password' do
+    expect { sign_up(password_confirmation: 'wrong_password')}.not_to change(User, :count)
+  end
+
+  scenario 'Gives an error message with a password that does not match' do
+  expect { sign_up(password_confirmation: 'wrong_password') }.not_to change(User, :count)
+  expect(current_path).to eq('/users/new')
+  expect(page).to have_content 'Password does not match the confirmation'
+end
```

Comment:
Lint/BlockAlignment: `end` at 20, 0 is not aligned with `scenario 'Gives an error message with a password that does not match' do` at 16, 2